### PR TITLE
FE2 terms

### PR DESCRIPTION
**Pull request by JR-Morgan**
Don't merge yet, this PR is destined for 2.19

### CODE REVIEW

File: Packages/systems.speckle.speckle-unity/Editor/Components/SpeckleReceiverEditor.cs [hunk 1]
```diff
@@ -21,7 +26,21 @@ public override async void OnInspectorGUI()
         {
             var speckleReceiver = (SpeckleReceiver)target;
 
-            DrawDefaultInspector();
+            //Selection
+            bool isFrontend2 = speckleReceiver.Account.Selected?.serverInfo.frontend2 ?? true;
+            EditorGUILayout.PropertyField(_accountSelection);
+            EditorGUILayout.PropertyField(
+                _streamSelection,
+                new GUIContent(isFrontend2 ? "Project" : "Stream")
```
Comment (oguzhankoral) [inline Packages/systems.speckle.speckle-unity/Editor/Components/SpeckleReceiverEditor.cs:34]: why we still need this switch? shouldn't we just use "Project"

Comment (JR-Morgan) [reply Packages/systems.speckle.speckle-unity/Editor/Components/SpeckleReceiverEditor.cs:34]: Good question,

This allows FE1 users to still see "Stream", "Branch", and "Commit" in the main UI, And FE2 users to see "Project", "Model", and  "Version".

I didn't intend to add it, but seeing as it was only only a handful lines, I decided to experiment.
I think its easy enough, might help avoid any confusion as users make the transition, and something we could kill in a few releases anyway.

What do you think? @bimgeek @oguzhankoral @clairekuang

Comment (oguzhankoral) [reply Packages/systems.speckle.speckle-unity/Editor/Components/SpeckleReceiverEditor.cs:34]: That was the case also in excel and I've changed it on my PR today, so users will always see new terms even if they have old account. I though this was a decision we made.

Comment (clairekuang) [reply Packages/systems.speckle.speckle-unity/Editor/Components/SpeckleReceiverEditor.cs:34]: I considered preserving this behavior for DUI2 as well, but the decision there is to just remove all old terminology. I think it's best to align and not show old terminology regardless of account

Comment (JR-Morgan) [reply Packages/systems.speckle.speckle-unity/Editor/Components/SpeckleReceiverEditor.cs:34]: Ok, I'll remove this switching.

**Pull request merged**